### ISSUE TITLE
Audio Stops on PantherCap and Truffles

### ISSUE THREAD

**Issue by patheros** [patheros/PathSetXOmriCohen#6]
From @omricohenmusic

> It also happened to me a few times that the audio stopped on PantherCap after changing the size and overlap, but I cannot reproduce it. The same with Truffles. Sometimes the first recording will not stay, but also this I cannot reproduce. I will let you know if I can find a way to reproduce these.

**Comment by patheros** [comment]
@omricohenmusic Do both channels stop, or just the right channel? I've had a similar issue where the right channel drops but haven't been able to reproduce it enough to fix it.

**Comment by omricohenmusic** [comment]
Yeah, true, sometimes it's only the right channel. I will test this a bit more and see if I can reproduce it.

**Comment by McGroggin** [comment]
I was about to submit a new report on Panther Cap but I believe what I am experiencing belongs here.

Any adjustment to the Time or Pitch parameters, even the slightest result in both output jacks going silent at zero volts. Sound would only come back when both knobs were set to 0.

Reloading the sample, trying a new sample, re-initializing the module etc did not resolve the issue. Restarting VCV did result in everything working as expected.

Panther Cap v2.02
VCV Rack 2 Pro 2.3.0
Win 10

**Comment by patheros** [comment]
@McGroggin yep that sounds like the same issue. Its been a persistent bug for awhile, but for me its only happens every so often. How often is this happening for you?

**Comment by Cubistguitar** [comment]
I am also experiencing audio stopping. I have often had the left channel only issues, but today both channels stopped and only reloading a mew Panther Cap and reloading audio would bring it back to usable. Just emptying and reloading audio would not help.
